### PR TITLE
cas-424 adds backend support for updating community threshold

### PR DESCRIPTION
**Pull request by rvmelkonian**
• updates model function and struct

### CODE REVIEW

**Review by dbslone — APPROVED** [review]

File: backend/main/models/community.go [hunk 1]
```diff
@@ -73,6 +72,12 @@ type UpdateCommunityRequestPayload struct {
 	Proposal_validation      *string     `json:"proposalValidation,omitempty"`
 	Proposal_threshold       *string     `json:"proposalThreshold,omitempty"`
 
+	//TODO dup fields in Community struct, make sub struct for both to use
+	Contract_name *string  `json:"contractName,omitempty"`
+	Contract_addr *string  `json:"contractAddr,omitempty"`
+	Public_path   *string  `json:"publicPath,omitempty"`
+	Threshold     *float64 `json:"threshold,omitempty"`
+
```
Comment (germanurrus) [inline backend/main/models/community.go:80]: I think the field `onlyAuthorsToSubmit` is missing

**Review by germanurrus — COMMENTED** [review]
Looks good, left a comment

File: backend/main/models/community.go (hunk 1 above)
Comment (germanurrus) [reply backend/main/models/community.go:80]: I also see that in the db there are two fields `proposal_threshold` and `threshold`, in the community creation we are using proposalThreshold: [here](https://github.com/DapperCollectives/CAST/blob/a7c7c9fe2b32feb11649aeba97d536147f0b375f/frontend/packages/client/src/hooks/useCommunity.js#L156)
but here I see `Threshold` being used which one is the correct one?

Comment (rvmelkonian) [reply backend/main/models/community.go:80]: `threshold` is being used.could you change the front end payload to always use `threshold?` on community creation?

**Pull request merged**